workflow/workflow.py

```diff
@@ -2353,6 +2356,21 @@ def update_available(self):
 
         return update_data['available']
 
+    @property
+    def prereleases(self):
+        """Should the workflow update to a newer pre-release version if
+        available?
+
+        :returns: ``True`` if pre-releases are enabled with the :ref:`magic
+        argument <magic-arguments>` or the ``update_settings`` dict, else
+        ``False``
+
+        """
+        if self._update_settings.get('prereleases'):
```

Comment:
Isn't the test the wrong way around? Shouldn't the user's settings override the defaults?

```python
if self.settings.get('__workflow_prereleases') is False:
    return False
return self.settings.get('__workflow_prereleases') or self._update_settings.get('prereleases')
```